extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -150,7 +150,11 @@ protected function saveToParserOutput( \ParserOutput $parserOutput, Nodes\NodeIn
 		// (see: PortableInfoboxDataService.class.php)
 		if ( $raw ) {
 			$infoboxes = json_decode( $parserOutput->getProperty( PortableInfoboxDataService::INFOBOXES_PROPERTY_NAME ), true );
-			$infoboxes[] = [ 'data' => $raw->getRenderData(), 'sourcelabels' => $raw->getSourceLabel() ];
+			$infoboxes[] = [
+				'data' => $raw->getRenderData(),
+				'sourcelabels' => $raw->getSourceLabel(),
```

Comment:
I'm wondering if merging those two into single "metadata" field would me more useful. Having an array containing objects with meta info about fields. What do you think? Would it help somehow or make it easier to use?